pages/content/amp-dev/documentation/guides-and-tutorials/develop/custom-javascript.md

```diff
@@ -3,120 +3,144 @@ $title: Use custom JavaScript in AMP pages
 $order: 7
 formats:
   - websites
-author: CrystalOnScript
+author:
+  - morss
 contributors:
+  - CrystalOnScript
   - fstanis
-description: For web experiences requiring a high amount of customization AMP has created amp-script, a component that allows the use of arbitrary JavaScript on your AMP page without affecting the page's overall performance.
+description: A guide to using amp-script, an AMP component that allows you to write custom JavaScript
 ---
 
-AMP strives to provide a consistently great experience to all users across the web by encouraging the use of high-functioning and seamless components that are ready to go out of the box.
+## Introduction
 
-Some web experiences require a high amount of customization that goes beyond the state binding capabilities of [`amp-bind`](../../../documentation/components/reference/amp-bind.md?format=websites) and the dynamic data retrieval and templating functionality of [`amp-list`](../../../documentation/components/reference/amp-list.md?format=websites) and [`amp-mustache`](../../../documentation/components/reference/amp-mustache.md?format=websites). For those one-off cases, AMP has created [`<amp-script>`](../../../documentation/components/reference/amp-script.md?format=websites), a component that allows the use of arbitrary JavaScript on your AMP page without affecting the page's overall performance.
+`amp-script` lets you write and run your own JavaScript in a way that maintains AMP's performance guarantees. Most AMP components enable common web interactions through their own logic, letting you build your page quickly without writing code or importing third-party libraries. By using `amp-script`, you can embrace custom logic for specific use cases or unique needs without losing AMP's benefits.
```

Comment:
I'm not sure we should say "without writing code" - it implies building with AMP is not coding.

Reply:
it's not realllllly coding... but you're right, this sounds bad. Changed.